public/js/list.class.js

```diff
@@ -1,18 +1,103 @@
-/*
-* Class List Declaration
-*/
-export class List{
-
-    addTitle(value){
-        this.title = value;
+"use strict";
```

Comment:
Ideally, you should not commit compiled js to git :) 

I would add them to `gitignore` or compile them into a different folder (maybe `dist`?) so it's easy to deal with this.
After compilation this js files are the ones you will deploy to final server. But this is not really interesting to have in git repository